wdl/workflows/references/make_references.wdl

```diff
@@ -0,0 +1,128 @@
+version 1.0
+
+
+import "../../subworkflows/build_bowtie_index.wdl" as bowtie
+import "../../subworkflows/build_bwa_index.wdl" as bwa
+import "../../subworkflows/build_fasta_index.wdl" as fai
+import "../../subworkflows/build_mappable_only_bed.wdl" as mappable
+import "../../subworkflows/get_center_sites.wdl" as center
+import "../../subworkflows/get_chrombuckets.wdl" as chr_buckets
+import "../../subworkflows/get_chrom_info.wdl" as chr_info
+import "../../subworkflows/get_chrom_sizes.wdl" as chr_sizes
+
+
+workflow make_references {
+    input {
+        BowtieIndex? bowtie_index
+        BwaIndex? bwa_index
+        File? center_sites
+        File? chrom_buckets
+        File? chrom_info
+        File? chrom_sizes
+        File? mappable_regions
+        File reference_fasta
+        Int mappability_kmer_length
+        String machine_size_bwa = "large"
```

Comment:
Should make ReferenceMachineSizes struct. Can look at structs/dnase for example.